src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/EditingPreferencesPane.java

```diff
@@ -121,6 +125,26 @@ public void onClick(ClickEvent event)
       lessSpaced(keyboardPanel);
       editingPanel.add(keyboardPanel);
 
+      HorizontalPanel projectPrefsPanel = new HorizontalPanel();
```

Comment:
Screenshot:

<img width="590" alt="image" src="https://user-images.githubusercontent.com/7817881/163249651-faaec2bc-9902-4838-9608-c6c2b80ce7f8.png">

Please note that Edit Project Preferences button does nothing when no project is activated. We should probably make this panel (text + button) or at least the button invisible when no project is activated

Reply:
> We should probably make this panel (text + button) or at least the button invisible when no project is activated

I agree this should only be shown if the user is currently working within a project.